spec/rspec/core/rake_task_spec.rb

```diff
@@ -66,6 +66,11 @@ def spec_command
         task.rspec_opts = "-Ifoo"
         expect(spec_command).to match(/#{task.rspec_path}.*-Ifoo/)
       end
+
+      it 'wont add --pattern if rspec_opts includes --pattern' do
```

Comment:
Also, this doc string is confusing.  It looks like it does show that it adds `--pattern` if `rspec_opts` includes `--pattern`.  How about:

``` ruby
it 'does not add the default --pattern if `rspec_opts` includes --pattern'
```